base/bucket_gocb.go

```diff
@@ -917,13 +923,13 @@ func (bucket CouchbaseBucketGoCB) GetWithXattr(k string, xattrKey string, rv int
 			// Successfully retrieved doc and (optionally) xattr.  Copy the contents into rv, xv and return.
 			contentErr := res.Content("", rv)
 			if contentErr != nil {
-				LogTo("gocb", "Unable to retrieve document content for key=%s, xattrKey=%s: %v", k, xattrKey, contentErr)
-				return false, contentErr, nil
+				LogTo("CRUD", "Unable to retrieve document content for key=%s, xattrKey=%s: %v", k, xattrKey, contentErr)
+				return false, contentErr, 0
```

Comment:
RetryLoop is already returning 'err, nil' if there's a non-recoverable error, so I don't believe any errors will be ignored.  I think it makes sense for the worker to return the appropriate type for their operation, even if the value needs to be defined as interface{} for reuse across multiple operation types.